lib/verify-vsce-auth.js

```diff
@@ -5,16 +5,29 @@ const execa = require('execa');
 
 module.exports = async (logger, cwd) => {
   logger.log('Verifying authentication for vsce');
+  logger.log(`VSCE_AZURE_CREDENTIALS: ${process.env.VSCE_AZURE_CREDENTIALS}`);
 
-  if (!process.env.VSCE_PAT) {
+  if (!process.env.VSCE_PAT && !process.env.VSCE_AZURE_CREDENTIALS) {
     throw new SemanticReleaseError(
-      'Empty vsce personal access token (`VSCE_PAT` environment variable) specified.',
-      'EEMPTYVSCEPAT',
+      'Neither vsce personal access token (`VSCE_PAT` environment variable) or Azure Credentials flag (`VSCE_AZURE_CREDENTIALS` environment variable) specified.',
+      'EVSCEAUTHNOTPROVIDED',
     );
   }
 
+  if (process.env.VSCE_PAT && process.env.VSCE_AZURE_CREDENTIALS) {
```

Comment:
```suggestion
  if ('VSCE_PAT' in process.env && 'VSCE_AZURE_CREDENTIALS' in process.env) {
```

And I think this check should come before the previous check, don't you think?